From b906a62119df4527b3278ad1db3707dba8134392 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Tue, 24 Jun 2008 13:46:32 +0800
Subject: [PATCH] ENGR00081543 align MX35 default config with other platform

Disable PCCARD, HWEVENT. Enable FM, Camera modules as "M".
Change partial settings for MMC and USB as "Y".

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 arch/arm/configs/imx35_3stack_defconfig |   65 ++++++++-----------------------
 1 files changed, 17 insertions(+), 48 deletions(-)

diff --git a/arch/arm/configs/imx35_3stack_defconfig b/arch/arm/configs/imx35_3stack_defconfig
index 7d3d0b2..f423250 100644
--- a/arch/arm/configs/imx35_3stack_defconfig
+++ b/arch/arm/configs/imx35_3stack_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.24
-# Tue May 20 02:33:31 2008
+# Tue Jun 24 10:44:08 2008
 #
 CONFIG_ARM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
@@ -157,6 +157,7 @@ CONFIG_ARCH_MXC=y
 # CONFIG_ARCH_MXC91321 is not set
 # CONFIG_ARCH_MX37 is not set
 CONFIG_ARCH_MX35=y
+# CONFIG_ARCH_MX51 is not set
 # CONFIG_ARCH_MX3 is not set
 # CONFIG_ARCH_MX27 is not set
 # CONFIG_ARCH_MX21 is not set
@@ -190,6 +191,7 @@ CONFIG_CPU_32=y
 # CONFIG_CPU_ARM926T is not set
 CONFIG_CPU_V6=y
 # CONFIG_CPU_32v6K is not set
+# CONFIG_CPU_V7 is not set
 CONFIG_CPU_32v6=y
 CONFIG_CPU_ABRT_EV6=y
 CONFIG_CPU_CACHE_V6=y
@@ -216,18 +218,7 @@ CONFIG_CACHE_L2X0=y
 CONFIG_ISA=y
 # CONFIG_PCI_SYSCALL is not set
 # CONFIG_ARCH_SUPPORTS_MSI is not set
-CONFIG_PCCARD=m
-# CONFIG_PCMCIA_DEBUG is not set
-CONFIG_PCMCIA=m
-CONFIG_PCMCIA_LOAD_CIS=y
-# CONFIG_PCMCIA_IOCTL is not set
-
-#
-# PC-card bridges
-#
-# CONFIG_I82365 is not set
-# CONFIG_TCIC is not set
-CONFIG_PCMCIA_PROBE=y
+# CONFIG_PCCARD is not set
 
 #
 # Kernel Features
@@ -509,7 +500,6 @@ CONFIG_BLK_DEV_LOOP=y
 # CONFIG_ATA_OVER_ETH is not set
 CONFIG_MISC_DEVICES=y
 # CONFIG_EEPROM_93CX6 is not set
-# CONFIG_IDE is not set
 
 #
 # SCSI device support
@@ -565,7 +555,6 @@ CONFIG_SCSI_LOWLEVEL=y
 # CONFIG_SCSI_SYM53C416 is not set
 # CONFIG_SCSI_T128 is not set
 # CONFIG_SCSI_DEBUG is not set
-# CONFIG_SCSI_LOWLEVEL_PCMCIA is not set
 # CONFIG_ATA is not set
 # CONFIG_MD is not set
 CONFIG_NETDEVICES=y
@@ -620,15 +609,6 @@ CONFIG_NET_PCI=y
 # CONFIG_USB_PEGASUS is not set
 # CONFIG_USB_RTL8150 is not set
 # CONFIG_USB_USBNET is not set
-CONFIG_NET_PCMCIA=y
-# CONFIG_PCMCIA_3C589 is not set
-# CONFIG_PCMCIA_3C574 is not set
-# CONFIG_PCMCIA_FMVJ18X is not set
-CONFIG_PCMCIA_PCNET=m
-# CONFIG_PCMCIA_NMCLAN is not set
-# CONFIG_PCMCIA_SMC91C92 is not set
-# CONFIG_PCMCIA_XIRC2PS is not set
-# CONFIG_PCMCIA_AXNET is not set
 # CONFIG_WAN is not set
 CONFIG_PPP=m
 # CONFIG_PPP_MULTILINK is not set
@@ -699,7 +679,7 @@ CONFIG_VT_CONSOLE=y
 CONFIG_HW_CONSOLE=y
 # CONFIG_VT_HW_CONSOLE_BINDING is not set
 # CONFIG_SERIAL_NONSTANDARD is not set
-# CONFIG_FM_SI4702 is not set
+CONFIG_FM_SI4702=m
 # CONFIG_MXC_MU is not set
 # CONFIG_MXC_SUPER_GEM is not set
 
@@ -723,13 +703,6 @@ CONFIG_HW_RANDOM=y
 # CONFIG_NVRAM is not set
 # CONFIG_DTLK is not set
 # CONFIG_R3964 is not set
-
-#
-# PCMCIA character devices
-#
-# CONFIG_SYNCLINK_CS is not set
-# CONFIG_CARDMAN_4000 is not set
-# CONFIG_CARDMAN_4040 is not set
 # CONFIG_RAW_DRIVER is not set
 # CONFIG_TCG_TPM is not set
 CONFIG_DEVPORT=y
@@ -773,6 +746,7 @@ CONFIG_I2C_MXC=y
 # CONFIG_I2C_DEBUG_ALGO is not set
 # CONFIG_I2C_DEBUG_BUS is not set
 # CONFIG_I2C_DEBUG_CHIP is not set
+# CONFIG_I2C_SLAVE is not set
 
 #
 # SPI support
@@ -842,7 +816,7 @@ CONFIG_VIDEO_CAPTURE_DRIVERS=y
 CONFIG_VIDEO_HELPER_CHIPS_AUTO=y
 # CONFIG_VIDEO_VIVI is not set
 # CONFIG_VIDEO_PMS is not set
-CONFIG_VIDEO_MXC_CAMERA=y
+CONFIG_VIDEO_MXC_CAMERA=m
 
 #
 # MXC Camera/V4L2 PRP Features support
@@ -850,9 +824,9 @@ CONFIG_VIDEO_MXC_CAMERA=y
 CONFIG_VIDEO_MXC_IPU_CAMERA=y
 # CONFIG_MXC_CAMERA_MC521DA is not set
 # CONFIG_MXC_CAMERA_MICRON111 is not set
-CONFIG_MXC_CAMERA_OV2640=y
-CONFIG_MXC_IPU_PRP_VF_SDC=y
-CONFIG_MXC_IPU_PRP_ENC=y
+CONFIG_MXC_CAMERA_OV2640=m
+CONFIG_MXC_IPU_PRP_VF_SDC=m
+CONFIG_MXC_IPU_PRP_ENC=m
 CONFIG_VIDEO_MXC_OUTPUT=y
 CONFIG_VIDEO_MXC_IPU_OUTPUT=y
 # CONFIG_VIDEO_MXC_OPL is not set
@@ -995,12 +969,6 @@ CONFIG_SND_MXC_SPDIF=m
 # CONFIG_SND_USB_CAIAQ is not set
 
 #
-# PCMCIA devices
-#
-# CONFIG_SND_VXPOCKET is not set
-# CONFIG_SND_PDAUDIOCF is not set
-
-#
 # System on Chip audio support
 #
 CONFIG_SND_SOC=y
@@ -1020,7 +988,7 @@ CONFIG_SND_SOC_AK4647=y
 #
 # CONFIG_SOUND_PRIME is not set
 CONFIG_HID_SUPPORT=y
-CONFIG_HID=m
+CONFIG_HID=y
 # CONFIG_HID_DEBUG is not set
 # CONFIG_HIDRAW is not set
 
@@ -1041,7 +1009,7 @@ CONFIG_USB_SUPPORT=y
 CONFIG_USB_ARCH_HAS_HCD=y
 # CONFIG_USB_ARCH_HAS_OHCI is not set
 CONFIG_USB_ARCH_HAS_EHCI=y
-CONFIG_USB=m
+CONFIG_USB=y
 # CONFIG_USB_DEBUG is not set
 
 #
@@ -1061,6 +1029,7 @@ CONFIG_USB_EHCI_HCD=m
 CONFIG_USB_EHCI_ARC=y
 CONFIG_USB_EHCI_ARC_H2=y
 CONFIG_USB_EHCI_ARC_OTG=y
+# CONFIG_USB_STATIC_IRAM is not set
 CONFIG_USB_EHCI_ARC_OTGHS=y
 # CONFIG_USB_EHCI_ARC_OTGFS is not set
 # CONFIG_USB_EHCI_SPLIT_ISO is not set
@@ -1083,7 +1052,7 @@ CONFIG_USB_EHCI_ROOT_HUB_TT=y
 #
 # may also be needed; see USB_STORAGE Help for more information
 #
-CONFIG_USB_STORAGE=m
+CONFIG_USB_STORAGE=y
 # CONFIG_USB_STORAGE_DEBUG is not set
 # CONFIG_USB_STORAGE_DATAFAB is not set
 # CONFIG_USB_STORAGE_FREECOM is not set
@@ -1190,14 +1159,14 @@ CONFIG_USB_G_SERIAL=m
 #
 # 
 #
-CONFIG_MMC=m
+CONFIG_MMC=y
 # CONFIG_MMC_DEBUG is not set
 # CONFIG_MMC_UNSAFE_RESUME is not set
 
 #
 # MMC/SD Card Drivers
 #
-CONFIG_MMC_BLOCK=m
+CONFIG_MMC_BLOCK=y
 CONFIG_MMC_BLOCK_BOUNCE=y
 # CONFIG_SDIO_UART is not set
 
@@ -1313,7 +1282,7 @@ CONFIG_MXC_MC9SDZ60_POWER=y
 #
 # MXC HARDWARE EVENT
 #
-CONFIG_MXC_HWEVENT=y
+# CONFIG_MXC_HWEVENT is not set
 
 #
 # MXC VPU(Video Processing Unit) support
-- 
1.5.4.4

